GPU/Common/DrawEngineCommon.cpp

```diff
@@ -501,7 +501,7 @@ bool DrawEngineCommon::ApplyFramebufferRead(bool *fboTexNeedsBind) {
 		lastFrameBlit = gpuStats.numFlips;
 	}
 	++blitsThisFrame;
-	if (blitsThisFrame > MAX_REASONABLE_BLITS_PER_FRAME * 2) {
+	if (blitsThisFrame > MAX_REASONABLE_BLITS_PER_FRAME * 4) {
```

Comment:
Should we just increase MAX_REASONABLE_BLITS_PER_FRAME?  It seems weird to multiply the constant, when this is its only use...

-[Unknown]

Reply:
I'm just gonna remove the whole thing, see #15515